ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,3 @@
+TAP_PACKAGE=1
+
+build() { :; }
```

Comment:
Why is build being disabled? If there is no Makefile to confuse things I would expect eupspkg to be able to do the right thing.